src/views/Onboarding/OnboardingSetup.vue

```diff
@@ -84,6 +88,19 @@ export default {
   padding: 0 !important;
   overflow: hidden;
 
+  &_description {
+    font-weight: 500;
+    font-size: 14px;
+  }
+
+  &_logo-wrap {
```

Comment:
Relative and absolute position should be avoided unless 100% necessary
In this case, you can adjust the margin of this element or elements below it to achieve what you need